setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='1.2.0',
+    version='2.0.0',
```

Comment:
If we're using semantic versioning, this is a backwards incompatible api change: http://semver.org/, which does require a major version change.  I'm not sure how strict we are about using that.  I hear people say we use it, but we have a lot of code that just hangs out in v1.